README.md

```diff
@@ -103,75 +76,27 @@ class YatLib {
     }
     ```
 
-3. Implement a function that initializes the library with the app-specific constants, user information and Yat records that should be linked to the Yat.
+3. Initialize the Yat library.
 
     ```kotlin
-    class MainActivity :
-        AppCompatActivity(),
-        YatLib.Delegate {
+    class MainActivity : AppCompatActivity(), YatIntegration.Delegate {
 
         // ...
 
         private fun initializeYatLib() {
-            // library configuration
-            val config = YatAppConfig(
-                name = "Super Cool Wallet",
-                sourceName = "SCW",
-                // used in partner-specific URLs
-                pathKey = "scw",
-                // app-specific public key
-                pubKey = "{64 character hex public key}",
-                // app-specific access code
-                code = "66b6a5aa-11b4-12a9-1c1e-84765ef174ab",
-                // app-specific authentication token
-                authToken = "AppToken84765783"
-            )
-            // Yat records
-            val yatRecords = listOf(
-                YatRecord(
-                    YatRecordType.ADA_ADDRESS,
-                    "DdzFFzCqrhsgwQmeWNBTsG8VjYunBLK9GNR93GSLTGj1FeMm8kFoby2cTHxEHBEraHQXmgTtFGz7fThjDRNNvwzcaw6fQdkYySBneRas"
-                ),
-                YatRecord(
-                    YatRecordType.DOT_ADDRESS,
-                    "GC8fuEZG4E5epGf5KGXtcDfvrc6HXE7GJ5YnbiqSpqdQYLg"
-                ),
-                YatRecord(
-                    YatRecordType.BTC_ADDRESS,
-                    "1NDyJtNTjmwk5xPNhjgAMu4HDHigtobu1s"
-                ),
-                YatRecord(
-                    YatRecordType.ETH_ADDRESS,
-                    "108dEFa0272dC118EF03a7993e4fC7A8AcF3a3d1"
-                ),
-                YatRecord(
-                    YatRecordType.XTR_ADDRESS,
-                    "d2e4db6dac593a9af36987a35676838ede4f69684ba433baeed68bce048e111b"
-                ),
-                YatRecord(
-                    YatRecordType.XMR_STANDARD_ADDRESS,
-                    "4AdUndXHHZ6cfufTMvppY6JwXNouMBzSkbLYfpAV5Usx3skxNgYeYTRj5UzqtReoS44qo9mtmXCqY45DJ852K5Jv2684Rge"
-                )
+   
+            val config = YatConfiguration(
+                appReturnLink = "app://y.at?action", // deep link to return to the app
+                organizationName = "Yat Labs",
+                organizationKey = "yat",
             )
-            // initialize the library
-            YatLib.initialize(
+   
+            YatIntegration.setup(
```

Comment:
Why do you changed the naming convention from YatLib to YatIntegration?

Reply:
@TruszczynskiA No, the change `YatLib` -> `YatIntegration` was there since one of the previous releases